pdns/dnsdist.cc

```diff
@@ -2564,24 +2517,17 @@ try
     }
   }
 
-  if(g_cmdLine.beDaemon) {
-    g_console=false;
-    daemonize();
-    writepid(g_cmdLine.pidfile);
-  }
-  else {
-    vinfolog("Running in the foreground");
-    warnlog("dnsdist %s comes with ABSOLUTELY NO WARRANTY. This is free software, and you are welcome to redistribute it according to the terms of the GPL version 2", VERSION);
-    vector<string> vec;
-    std::string acls;
-    g_ACL.getLocal()->toStringVector(&vec);
-    for(const auto& s : vec) {
-      if (!acls.empty())
-        acls += ", ";
-      acls += s;
-    }
-    infolog("ACL allowing queries from: %s", acls.c_str());
+  vinfolog("Running in the foreground");
```

Comment:
nit: not sure this message is still relevant

Reply:
Oh right